view-user/src/pages/test/index.tsx

```diff
@@ -27,11 +32,24 @@ const testBingoPrize = {
   prizeImage: testPrizeImage,
 };
 
+const testBingoNumbers: BingoNumber[] = [
+  { id: 1, number: 1, createdAt: "2024-08-01", updatedAt: "2024-08-01" },
+  { id: 2, number: 2, createdAt: "2024-08-01", updatedAt: "2024-08-01" },
+  { id: 3, number: 3, createdAt: "2024-08-01", updatedAt: "2024-08-01" },
+  { id: 4, number: 4, createdAt: "2024-08-01", updatedAt: "2024-08-01" },
+  { id: 5, number: 5, createdAt: "2024-08-01", updatedAt: "2024-08-01" },
+  { id: 6, number: 6, createdAt: "2024-08-01", updatedAt: "2024-08-01" },
+  { id: 7, number: 7, createdAt: "2024-08-01", updatedAt: "2024-08-01" },
+  { id: 8, number: 8, createdAt: "2024-08-01", updatedAt: "2024-08-01" },
+  { id: 9, number: 9, createdAt: "2024-08-01", updatedAt: "2024-08-01" },
+];
+
 const HomePage: React.FC = () => {
   return (
     <div>
       {/* <NumberCardSmall BingoNumber={testBingoNumber}></NumberCardSmall> */}
-      <PrizeCard BingoPrize={testBingoPrize}></PrizeCard>
+      {/* <PrizeCard BingoPrize={testBingoPrize}></PrizeCard> */}
+      <NumberCardList firstNumber={false} bingoNumber={testBingoNumbers} />
```

Comment:
```suggestion
      <NumberCardList firstNumber bingoNumber={testBingoNumbers} />
```
明示的に指定しなくても大丈夫です！
firstNumberと書いたらtrueになって書かなかったらfalseになります～